package.json

```diff
@@ -23,7 +24,7 @@
     "body-parser": "^1.19.0",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "mongoose": "^5.10.9",
+    "mongoose": "^5.12.0",
```

Comment:
Changing the dependency version without consulting is a serious no no. This can lead to a lot of errors if the rest of the code was built on an older version of mongoose and there have been major changes in the future versions, which is the case here.